sdk/monitor/Microsoft.OpenTelemetry.Exporter.AzureMonitor/tests/Microsoft.OpenTelemetry.Exporter.AzureMonitor.Integration.Tests/TelemetryItemTests.cs

```diff
@@ -67,13 +63,14 @@ public void VerifyActivity(ActivityKind activityKind)
                 });
         }
 
-        // TODO: ADD THIS TEST AFTER IMPLEMENTING ILOGGER EXPORTER
-        //public void VerifyILoggerAsTelemetryItem
-
-        private void Setup(out BatchExportProcessor<Activity> processor, out MockTransmitter transmitter )
+        private TelemetryItem RunActivityTest(Action<ActivitySource> testScenario)
         {
-            transmitter = new MockTransmitter();
-            processor = new BatchExportProcessor<Activity>(new AzureMonitorTraceExporter(
+            // SETUP
+            var ActivitySourceName = "MyCompany.MyProduct.MyLibrary";
+            var activitySource = new ActivitySource(ActivitySourceName);
```

Comment:
using var activitySource or calling activitySource.dispose in clean up will help dispose the object.